tests/acceptance/test_create_project.py

```diff
@@ -31,9 +32,26 @@ def test_no_teams(self):
     def test_many_teams(self):
         self.team = self.create_team(organization=self.org, name="Mariachi Band")
         self.team2 = self.create_team(organization=self.org, name="team two")
-        self.create_member(
-            user=self.user, organization=self.org, role="owner", teams=[self.team, self.team2]
-        )
+
         self.browser.get(self.path)
         self.browser.wait_until_not(".loading")
         self.browser.snapshot(name="create project many teams")
+
+    def test_select_correct_platform(self):
+        self.create_team(organization=self.org, name="team three")
+
+        self.browser.get(self.path)
+        self.browser.wait_until_not(".loading")
+
+        self.browser.click('[data-test-id="platform-javascript-react"]')
+        self.browser.wait_until_not(".loading")
+        self.browser.click('[data-test-id="create-project"]')
+
+        self.browser.wait_until_not(".loading")
+        self.browser.wait_until("h2")
+
+        title = self.browser.find_element(by=By.CSS_SELECTOR, value="h2")
+        info_box = self.browser.find_element(by=By.CSS_SELECTOR, value="div.ref-info > span")
```

Comment:
I think if we just check the title it will be enough because `div.ref-info > span` might change soon